### ISSUE TITLE
Accessibility: Product gallery images on single product contain title attribute, resulting in unnecessary noise for screen reader users

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43576]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

On the single product page, the product images in the gallery output a title attribute. Here's an example:
<img width="1534" alt="Screenshot 2024-01-12 at 11 14 54 AM" src="https://github.com/woocommerce/woocommerce/assets/13153456/b802c74d-ebbc-4e5c-8397-5a3fd0b8878d">

When a screen reader encounters these images it reads out both the alt text and the title attribute. 

In the worse-case scenario, the title is the file name (as is shown in the screenshot) and junky information is read out. This is most commonly the case as WordPress creates the title from the file name on upload. In a best-case scenario, the user has written a title, but it is commonly duplicative of what is in the alt text.

The title attribute should not be included in image tags.

### Expected behavior

Images will not have a title attribute and use the alt attribute only.

### Actual behavior

A title attribute is present.

### Steps to reproduce

Go to a product single and inspect the images in the image gallery.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by roykho** [comment]
@amberhinds - Thanks for reporting this. We'll have a look when available.

**Comment by amesplant** [comment]
I am looking into this issue.
